tenable_io/api/sc_reports.py

```diff
@@ -0,0 +1,46 @@
+from tenable_io.api.base import BaseApi
+from tenable_io.api.models import ScReport
+
+
+class ScReportsApi(BaseApi):
+
+    def show(self, container_id):
+        """Get JSON format report using container ID.
+
+        :param container_id: The ID of the container image.
+        :raise TenableIOApiException:  When API error is encountered.
+        :return: An instance of :class"`tenable_io.api.models.ScReport`.
```

Comment:
You have an extra `"` here? after `class`, there are more of these typos in the file as well.